.gitignore

```diff
@@ -22,3 +22,4 @@ hs_err_pid*
 .project
 .settings/
 /bin/
+.vscode/
```

Comment:
Do we need to gitignore .vscode?

Reply:
It's something which users will have IDE specific. So added it just to make sure that it doesn't get pushed.

Reply:
Resolving...